dialects/linux/dproc.c

```diff
@@ -1518,12 +1522,71 @@ process_proc_map(p, s, ss)
 		efs = sv = 1;
 	    else
 		efs = 0;
-	    if (!efs) {
-		if (HasNFS)
-		    sv = statsafely(fp[6], &sb);
-		else
-		    sv = stat(fp[6], &sb);
-	    }
+
+	/*
```

Comment:
Align comments to code

Reply:
I am unsure about the code style, @masatake

Reply:
The style is not well defined. However, if the lines you changed are not well aligned with the other code around you changed ON GITHUB WEB INTERFACE, I would you to adjust your changes. In the other words, keep your change as if the change has been there for years.